tests/integration/conftest.py

```diff
@@ -135,3 +139,14 @@ def ingress_load_balancer_url() -> str:
         raise RuntimeError(msg)
     except Exception as e:
         raise RuntimeError() from e
+
+
+@fixture(scope="session")
+def setup_cluster(request):
```

Comment:
Do you mind adding a type - hint? 

Use the search on the pytest docs to dig it up. This is new functionality for me and I have no ideas what this thing is 🙂

Reply:
Sure, I admit it's not obvious at all where it comes from.